cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -47,6 +47,9 @@ spec:
             {{ if eq .Cluster.ConfigItems.fabric_gateway_controller_allow_all_filters "true" }}
             - -allow-all-filters=true
             {{ end }}
+            {{ if eq .Cluster.ConfigItems.fabric_gateway_controller_enable_versioning "true" }}
+            - -enable-versioning
```

Comment:
=true?

Reply:
`-enable-versioning` and `-enable-versioning=true` work the same way.

Reply:
I think explicit is better is this case like `-allow-all-filters=true` but that's a minor